qiskit/pulse/transforms/alignments.py

```diff
@@ -48,17 +48,15 @@ def align(self, schedule: Schedule) -> Schedule:
 
     def to_dict(self) -> Dict[str, Any]:
```

Comment:
This is a bit confusing. Should this not be an abstract method now?

Reply:
This is deprecated and now it implements `context_params` property for parameter assignment and context serialization. This is the same idea as `Instruction` implements `.operands` property.